joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 250

egm_state_controller:
  type         : abb_egm_state_controller/EGMStateController
  publish_rate : 250

joint_trajectory_controller:
  type: velocity_controllers/JointTrajectoryController
  joints:
    - joint_1
    - joint_2
    - joint_3
    - joint_4
    - joint_5
    - joint_6
  gains:
    joint_1: {p: 1.0, i: 0.0, d: 0.1, i_clamp: 0.0}
    joint_2: {p: 1.0, i: 0.0, d: 0.1, i_clamp: 0.0}
    joint_3: {p: 1.0, i: 0.0, d: 0.1, i_clamp: 0.0} # was 1.0, 0.0, 0.1, 0.0
    joint_4: {p: 1.0, i: 0.0, d: 0.1, i_clamp: 0.0}
    joint_5: {p: 1.0, i: 0.0, d: 0.1, i_clamp: 0.0}
    joint_6: {p: 1.0, i: 0.0, d: 0.1, i_clamp: 0.0}
  state_publish_rate: 250
  action_monitor_rate: 10
  stop_trajectory_duration: 0.0

  constraints:
    goal_time: 1.2 # was 0.5
    stopped_velocity_tolerance: 0.02
    joint_1: {trajectory: 0.12, goal: 0.05} # was 0.12, 0.05
    joint_2: {trajectory: 0.12, goal: 0.05}
    joint_3: {trajectory: 0.12, goal: 0.05}
    joint_4: {trajectory: 0.12, goal: 0.05}
    joint_5: {trajectory: 0.12, goal: 0.05}
    joint_6: {trajectory: 0.12, goal: 0.05}